pkg/api/v1/conversion.go

```diff
@@ -44,6 +44,7 @@ func addConversionFuncs() {
 			case "metadata.name",
 				"metadata.namespace",
 				"status.phase",
+				"status.podIP",
```

Comment:
Not that I'm opposed, but this isn't strictly needed for the downward API is it?